jobs/user-vms/admin-exams.hcl

```diff
@@ -0,0 +1,229 @@
+job "admin-exams" {
+  datacenters = ["aperture"]
+
+  group "ayden-vm" {
+
+    network {
+      mode = "host"
+    }
+
+    service {
+      name = "ayden-vm"
+    }
+
+    task "ayden-vm" {
+      constraint {
+        attribute = "${attr.unique.hostname}"
+        value = "chell"
+      }
+
+      resources {
+        cpu = 12000
+        memory = 4096
+      }
+
+      artifact {
+        source = "http://136.206.16.5:8000/base-images/debian-12-genericcloud-amd64-30G.qcow2"
+        destination = "local/ayden-vm.qcow2"
+        mode = "file"
+      }
+
+      driver = "qemu"
+
+      config {
+        image_path = "local/ayden-vm.qcow2"
+        accelerator = "kvm"
+        drive_interface = "virtio"
+
+        args = [
+          "-netdev",
+          "bridge,id=hn0",
+          "-device",
+          "virtio-net-pci,netdev=hn0,id=nic1,mac=52:54:84:ba:49:20",
+          "-smbios",
+          "type=1,serial=ds=nocloud-net;s=http://136.206.16.5:8000/ayden-vm/",
+        ]
+      }
+    }
+  }
+  group "arkues-vm" {
```

Comment:
I think we can generalise this a little, no need to commit every user. Perhaps commit users who have passed the exam?

Reply:
Cleaned this up, as well as adding some comments for clarification